tools/parse_module_type.py

```diff
@@ -258,31 +258,45 @@ def parseWebsite(autoDownload=True):
     modules_with_type['PIPE'] = projektmodul
 
     # ICS modules
-    ics_modules_with_type['DB&S'] = kernmodul
-    ics_modules_with_type['NETW1'] = kernmodul
-    ics_modules_with_type['NETW2'] = kernmodul
+    ics_modules_with_type['ADRM'] = kernmodul
     ics_modules_with_type['ADS'] = kernmodul
-    ics_modules_with_type['INTROL'] = kernmodul
-    ics_modules_with_type['SPTA'] = kernmodul
-    ics_modules_with_type['SPREN1'] = kernmodul
-    ics_modules_with_type['ISLAB_K'] = kernmodul
     ics_modules_with_type['ANLS'] = kernmodul
-    ics_modules_with_type['SPTA'] = kernmodul
+    ics_modules_with_type['CISO ISSUES'] = majormodul
     ics_modules_with_type['CRS'] = erweiterungsmodul
+    ics_modules_with_type['DB&S'] = kernmodul
     ics_modules_with_type['DSO'] = kernmodul
-    ics_modules_with_type['PRIVACY1'] = kernmodul
-    ics_modules_with_type['OSA'] = kernmodul
+    ics_modules_with_type['ENGPRA'] = zusatzmodul
+    ics_modules_with_type['HOA'] = majormodul
+    ics_modules_with_type['IAM'] = majormodul
+    ics_modules_with_type['INTROL'] = kernmodul
+    ics_modules_with_type['IRFORENSIC'] = erweiterungsmodul # as well as kernmodul
```

Comment:
How can a module be an "erweiterungsmodul" and a "kernmodul"?

Reply:
I dunno ¯\\\_(ツ)_/¯. HSLU😅 . 

I'm guessing different curriculums for different years. 

[(Module Spec)](https://mycampus.hslu.ch/-/media/campus/common/files/dokumente/other/mycampus/i/dokumente-ausbildung/bachelor/modulbeschriebe-ics/angewandte-incident-response-und-it-forensik.pdf?la=de-ch)

Reply:
Ah It's probably a "kernmodule" for the forencis major and "erweiterungsmodul" for all other majors.

My suggestion is to categorize it as "erweiterungsmodul" and handle different majors in a follow up. WDYT?

Reply:
Oh god, I didn't know that there are modules now, with multiple types already. I guess I shouldn't be surprised  by any strange things our esteemed HSLU does any more. But "erweiterungsmodul" sounds fine, as we can't handle majors at the moment.

Reply:
Yeah, it would probably need a user input in the extension as well. I don't think we have the info which major a user does on the "Meine Anmeldungen" page or anywhere else..